server.go

```diff
@@ -486,49 +499,83 @@ func NewFromConfig(config ServerConfig) (*Server, error) {
 	// found during setup we should use logger.Fatal or
 	// logger.Panic, because that will give us breakage monitoring
 	// through Sentry.
-	numWorkers := 1
-	if conf.NumWorkers > 1 {
-		numWorkers = conf.NumWorkers
-	}
-	logger.WithField("number", numWorkers).Info("Preparing workers")
-	// Allocate the slice, we'll fill it with workers later.
-	ret.Workers = make([]*Worker, numWorkers)
+
 	ret.numReaders = conf.NumReaders
 
 	// This must come before worker initialization. We need to
 	// initialize workers with state from *Server.IsWorker.
 	ret.ForwardAddr = conf.ForwardAddress
 
-	// Control whether Veneur should emit metric
-	// "veneur.flush.unique_timeseries_total", which may come at a
-	// slight performance hit to workers.
-	ret.CountUniqueTimeseries = conf.CountUniqueTimeseries
```

Comment:
This PR unships this experimental feature because it would be additional work to refactor this to support WorkerSets, and because it may be superceded by https://github.com/stripe/veneur/pull/858. I think it's worth discussing the implications of this so that we make changes with intentionality here.